package.json

```diff
@@ -74,6 +74,7 @@
     "@babel/preset-flow": "^7.12.1",
     "@babel/preset-react": "^7.12.1",
     "@babel/runtime": "^7.12.5",
+    "@miherlosev/esm": "3.2.26",
     "@types/node": "^12.20.10",
```

Comment:
Could you provide more details to your discussion? In light of the recent npm libraries 'colors' and 'faker' sabotage, it would feel more confident to have dependencies managed by organisations. @miherlosev @AndreyBelym 

I also noticed you hadn't pushed the source code for the latest version to your fork - please do so.

Reply:
@miherlosev Yes, at a minimum, could we get your source code modifications to `esm` pushed into your forked repo for visibility? Thank you in advance.